test/unit/carrier_test.rb

```diff
@@ -84,6 +84,24 @@ def test_timestamp_from_business_day_returns_fifteen_days_in_the_future
     end
   end
 
+  def test_timestamp_from_business_day_handles_saturday
+    current = DateTime.new(2016, 7, 9) # Saturday
+    expected = DateTime.new(2016, 7, 11)
+
+    Timecop.freeze(current) do
+      assert_equal expected, @carrier.send(:timestamp_from_business_day, 1)
+    end
+  end
+
+  def test_timestamp_from_business_day_handles_saturday
```

Comment:
It should fail and complain about duplicate test existing already.

Also +1 to converting everything to `test "" do`. Doing it on a file-by-file basis would probably be more manageable.